### ISSUE TITLE
Feature Request: Converge/diverge several verticies to their center point

### ISSUE THREAD

**Issue by Gamepro5** [blackears/cyclopsLevelBuilder#142]
![image](https://github.com/blackears/cyclopsLevelBuilder/assets/23440295/f2b8e83e-cb84-4bd1-a045-9e3eaeb5f0bb)
For example, If I select the top vertices on a cylinder, using the scroll wheel up should move them farther away from the center point (where the gizmo is) and using scroll wheel down should bring them closer to the central point. If I scroll down it would make a cone from the cylinder.

The direction that an individual vertex is to move is determined from the point in space that the individual vertex is on and the central point.

**Comment by blackears** [comment]
I don't think this would work.  The mouse wheel is already used for zooming the camera.  I do want to add a scaling gizmo eventually, but that will have to wait on a few other things.

**Comment by Gamepro5** [comment]
> The mouse wheel is already used for zooming the camera.

This shouldn't be a problem. You've already overloaded the functionality for the scroll wheel when it came to drawing cylinders so I see no reason why it can't be done again. Worst case it can be done in combination with holding a specific key that isn't used by Godot.

**Comment by blackears** [comment]
A command for merging vertices at their center has been added.

**Comment by Gamepro5** [comment]
Does it allow you to partially merge them? I wanted to create an elevated control point

**Comment by blackears** [comment]
Not at the moment.  It will take a lot more dev work to get to the point where that is possible.
